.github/workflows/tests_linux_python.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       matrix:
         os: [ubuntu-22.04]
         pl_backend: ["lightning_qubit"]
-    timeout-minutes: 60
+    timeout-minutes: 75
```

Comment:
Likely this may be influenced by @vincentmr 's PR https://github.com/PennyLaneAI/pennylane-lightning/pull/696
Do we need this timelimit set this high once that one goes in?

Reply:
For the Python test with kokkos, yes. I also incremented the other ones in this file to be 100% sure that we have a green CI

Reply:
Let's revisit this then after that above PR goes in. 75 mins here is a lot